### DISCUSSION TITLE
issue with intel mac rendering

### DISCUSSION THREAD

**Discussion by ROYALKINGSJ** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#460] [General]
i have mac os montenary and the radeon pro render is working like shit......
It does not render the image, it does not show the render preview in viewport and it freezes blender and eventually crashes.....
i have 16GB ram and amd radeon pro 5500m and blender 3.0.0 stable

**Comment by bsavery** [comment]
Works fine here.  File an issue above and attach a scene and any logs that you can?
